apps/topics/models/schema.py

```diff
@@ -0,0 +1,40 @@
+import graphene
+from graphene_django.types import DjangoObjectType, ObjectType
+
+from apps.topics.models.topic import Topic
+
+
+# GraphQL type for Topic model
+class TopicType(DjangoObjectType):
+    pk = graphene.Int(source='pk')  # adding private key to the fields
+
+    class Meta:
+        model = Topic
+        fields = (
+            'key',
+            'pk',
+        )
+
+
+# GraphQL query for all Topics with the fields of 'key' and 'pk'
+class Query(graphene.ObjectType):
+    all_topics = graphene.List(TopicType)
+
+    def resolve_all_topics(self, info, **kwargs):
+        return Topic.objects.all()
+
+
+schema = graphene.Schema(query=Query)
+
+
+def query_result():
```

Comment:
Is this function only for testing purposes? My understanding is that we would want all request schemas defined on the frontend, so that control over what data is retrieved is always held by the frontend. Is that correct?

Reply:
Good point @mattfredericksen.

@aaw0151, I just looked at a couple tutorials, and it looks like most developers use Apollo to interact with GraphQL APIs from a React page.

This is the best walkthrough I found: https://zoejoyuliao.medium.com/django-graphql-react-2-integrate-graphql-into-your-react-project-71fa74f1cb00

Reply:
The schema will be updated in the next implementation to a more "generic" schema to allow for easier additions.